hawc/apps/summary/models.py

```diff
@@ -239,6 +242,8 @@ def build_default(cls, assessment_id: int, table_type: int) -> "SummaryTable":
         instance = cls(assessment_id=assessment_id, table_type=table_type)
         schema = instance.get_content_schema_class()
         instance.content = schema.get_default_props()
+        if "assessment_id" in instance.content:
```

Comment:
This allows a default to be given when it is assessment specific. Note the default for the study outcome table is pretty much empty since it relies a lot on other ids (study/metric), but at least it still follows the pattern.

Reply:
Same as above, we should remove this from the content and just rely on `instance.assessment_id`